hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -121,28 +121,32 @@
       milestone: V2
       tests: []
     }
-
-    // SPI_HOST (pre-verified IP) integration tests:
     {
-      name: chip_spi_host_tx_rx
-      desc: '''Verify the transmission of data on the chip's SPI host port.
-
-             Details TBD. Run with min and max SPI clk frequencies. Also, run with single, dual,
-             and quad SPI modes.
+      name: chip_spi_device_pass_through
+      desc: '''Verifies the pass through mode from an end-to-end perspective.
 
+            SW configures the SPI device and host in pass through mode. The testbench sends a random
+            payload over the SPI device interface and verifies its integrity on the SPI host
+            interface. Run with min and max SPI clk frequencies. Also, run with single, dual,
+            and quad SPI modes. Details TBD.
```

Comment:
You may also want to mention the pass through data filtering mechanism here for completeness.

Reply:
Done - I added that as as separate test. Note that this is SPI device specific, which was not the focus of this review session.